client/src/pages/Landing.js

```diff
@@ -0,0 +1,162 @@
+  },
+  errors: {
+    color: 'red'
+  }
+})
+
+export default function Landing() {
+  const classes = useStyles();
+
+  const [open, setOpen] = useState(false);
+
+  const [errorName, setErrorName] = useState('');
+  const [errorEmail, setErrorEmail] = useState('');
+  const [errorPassword, setErrorPassword] = useState('');
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [language, setLanguage] = useState('');
+  
+  const [errorLanguage, setErrorLanguage] = useState('')
+
+  // Npm email-validator is acting up; if anyone has a better email validation function
+  // feel free to replace the function below
+  const isEmail = email => /^\S+@\S+$/.test(email)
+
+  const handleSubmit = () => {
+    if (isEmail(email) && password.length >= 6 && language) {
+      axios.post(`${baseURL}/user/signup`, {
+        name, // name: name (shorthand)
+        email,
+        password,
+        primaryLanguage: language
+      }).then(response => {
+        console.log(response.data)
+        // if (response.status === 200 || response.status === 201){
+  
+        // }
+      })
+    } else {
+      // one or both or all four may happen, hence the if statement structure. Also disable errors once criteria is met.
+      if (!name) setErrorName('Name required.')
+      if (password.length < 6) setErrorPassword('Password must be at least 6 characters.');
+      if (!isEmail(email)) setErrorEmail('Invalid email.');
+      if (!language) setErrorLanguage('Please select a language.')
+      if (name) setErrorName('')
+      if (password.length >= 6) setErrorPassword('');
+      if (isEmail(email)) setErrorEmail('')
+      if (language) setErrorLanguage('')
+    }
+  }
+
+  return (
+    <Box display="flex">
+      {/** The left side: Image saying "Converse with anyone with any language" */}
+      <Box >
+        <img src={Background} className={classes.image} alt="People talking and texting, with purpose of the web app."/>
+      </Box>
+
+      {/** The right side, the sign up */}
+      <Box className={classes.outerMargins} >
+        <Box display="flex" className={classes.marginBottom50}>
+          <Typography variant="h5">Already have an account? </Typography>
+          <Link to="/login">
+            <Button variant="outlined" color="primary" className={classes.marginLeft10}>
+              Login
+            </Button>
+          </Link>
+          
+        </Box>
+        
+        <Typography 
+          variant="h4" 
+          fontWeight="fontWeightBold"
+          className={classes.marginBottom5}
+        >Create an Account.</Typography>
+        <Box display="flex" flexDirection="column">
+          <TextField 
+            label="Name" 
+            onChange={(event) => setName(event.target.value)}
+            className={classes.marginBottom5}
+          />
+          <Typography variant="h6" className={classes.errors}>{errorName}</Typography>
```

Comment:
This can wait until a future PR, but we can use TextField's error and helperText here instead: https://material-ui.com/components/text-fields/#validation